extra-files/cloud_setup/README.md

```diff
@@ -39,12 +39,10 @@ This tool uses `tool_panel_section_label` as we are making a new section in the
   install_resolver_dependencies: True
 ```
 
- * `dependency_resolvers_conf.xml` : Modified `dependency_resolvers_conf` file with conda moved up as the first option.
-
  * `data_manager_tool_list.yml` : List of data managers installed.
 
  * `data_manager_tasks.yml` : Run data manager tasks with dbkeys (organisms).
 
  * `fix_hostname.sh` : Custom script to fix the hostname on Jetstream instances.
 
- *  `tool_sheds_conf.xml` : Two tool sheds are available right now through this file, if the user has a local toolshed, they can add it to this list.
+ * `tool_sheds_conf.xml` : Two tool sheds are available right now through this file, if the user has a local ToolShed, they can add it to this list.
```

Comment:
Two tool sheds are available right now through this file (Galaxy main and test toolsheds). Users can reference other toolsheds in the file.